Scripts/RemediationScripts/Remediate-DisableRemoteDebuggingForAppServices.ps1

```diff
@@ -167,23 +172,36 @@ function Disable-RemoteDebuggingForAppServices
 
         [String]
         [Parameter(ParameterSetName = "WetRun", HelpMessage="Specifies the path to the file to be used as input for the remediation")]
-        $FilePath
+        $FilePath,
+
+        [String]
+        $Path,
+
+        [Switch]
+        $AutoRemediation,
+
+        [String]
+        $TimeStamp
```

Comment:
same as above.

Reply:
AutoRemediate make more sense true.

Reply:
But AutoRemediation also makes sense. Will keep it open and change last as it's 'find and replace' kind of thing